Dockerfile.neuron

```diff
@@ -20,11 +20,15 @@ ARG APP_MOUNT=/workspace
 VOLUME [ ${APP_MOUNT} ]
 WORKDIR ${APP_MOUNT}/vllm
 
-RUN python3 -m pip install --upgrade pip
-RUN python3 -m pip install --no-cache-dir fastapi ninja tokenizers pandas
-RUN python3 -m pip install sentencepiece transformers==4.45.2 -U
-RUN python3 -m pip install neuronx-cc==2.16.345.0 --extra-index-url=https://pip.repos.neuron.amazonaws.com -U
-RUN python3 -m pip install pytest
+COPY requirements-build.txt /app/vllm/
+COPY requirements-torch-neuron.txt /app/vllm/
+COPY requirements-neuron.txt /app/vllm/
+COPY requirements-common.txt /app/vllm/
```

Comment:
Is this location correct? Would it not be better to copy these into `WORKDIR`?

Reply:
fixed